packages/google_maps_flutter/google_maps_flutter_web/lib/src/map_styler.dart

```diff
@@ -0,0 +1,40 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:js_interop';
+
+/// The interop type for a Google Maps Map Styler.
+///
+/// See: https://developers.google.com/maps/documentation/javascript/style-reference#stylers
+@JS()
+@staticInterop
```

Comment:
nits: You don't need `@staticInterop` or `@anonymous` anywhere with extension types.

Reply:
Handled in https://github.com/flutter/packages/pull/6216.